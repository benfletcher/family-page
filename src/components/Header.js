import React from 'react';

const Header = () => (
  <div className="navButton">
    <p className="navTitle">Gallery
      <a href="www.upload.com">
        <img className="uploadButton" alt="upload" src="upload.png" />
      </a>
    </p>
  </div>
);

export default Header;
