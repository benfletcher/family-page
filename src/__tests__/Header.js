/* eslint-env browser, jest */
import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

test('Header snapshot is the same', () => {
  const component = renderer.create(
    <Header />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
