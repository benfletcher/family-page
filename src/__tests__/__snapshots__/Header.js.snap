exports[`test Header snapshot is the same 1`] = `
<div
  className="navButton">
  <p
    className="navTitle">
    <a
      onClick={[Function]}
      style={Object {}}>
      Home
    </a>
    <span />
    <a
      onClick={[Function]}
      style={Object {}}>
      Gallery
    </a>
    <a
      href="http://localhost:8080/auth/logout">
      Logout
    </a>
    <a
      onClick={[Function]}
      style={Object {}}>
      <i
        aria-hidden="true"
        className="fa fa-cloud-upload fa-2x uploadButton" />
    </a>
  </p>
</div>
`;
